events/smart-descriptions.json

```
@@ -840,10 +840,146 @@
       ]
     },
     {
-      "value": "Process {process.executable} created by {user.name} on {log.hostname}",
+      "value": "Process {process.executable} changed the creation time of the file {file.name} on {log.hostname}",
       "conditions": [{
           "field": "action.id",
-          "value": 1
+          "value": 2
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Network connection from {source.ip} to {destination.ip} on port {destination.port} by {process.executable}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 3
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Process {process.executable} terminated on {log.hostname}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 5
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Driver {process.executable} loaded on {log.hostname}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 6
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Process {action.properties.SourceImage} created a thread in process {action.properties.TargetImage} on {log.hostname}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 8
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Process {process.executable} read on {action.properties.Device} device on {log.hostname} hostname",
+      "conditions": [{
+          "field": "action.id",
+          "value": 9
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "File {file.name} created by {process.executable} on {log.hostname}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 11
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Registry key {action.properties.TargetObject} created by {process.executable} on {log.hostname}",
+      "conditions": [{
+          "field": "action.id",
+          "value": 12
+        },
+        {
+          "field": "action.properties.MessEventType",
+          "value": "CreateKey"
+        },
+        {
+          "field": "event.provider",
+          "value": "Microsoft-Windows-Sysmon"
+        }
+      ]
+    },
+    {
+      "value": "Registry value of the key {action.properties.TargetObject} created by {process.executable} on {log.hostname}",
```

```suggestion
      "value": "Registry value {action.properties.TargetObject} created by {process.executable} on {log.hostname}",
```

In this case `TargetObject` is the path to the value, not the key.